ui/shared/utils/semanticUtils.js

```diff
@@ -0,0 +1,35 @@
+// Utility functions for Semantic UI
```

Comment:
These are not utilities related to semantic UI, they are utilities related to a component that is named `BaseSemanticInput` that is a form input wrapper component which handles one out of the many semantic ui classes. I don't think these really belong in their own file, I think they belong in `shared/components/form/Inputs`, but if you really prefer to have them in their own file it should be in the form folder and named accordingly - maybe `shared/components/form/formInputUtils.js`

Reply:
I've moved them to `shared/components/form/Inputs` and deleted `semanticUtils.js` 👍